.github/workflows/build-and-test.yaml

```diff
@@ -1,12 +1,16 @@
-name: build-and-test-scheduled
+name: build-and-test
 
 on:
+  push:
+    branches:
+      - main
+      - tier4/proposal
   schedule:
     - cron: 0 19 * * * # run at 4 AM JST
   workflow_dispatch:
 
 jobs:
-  build-and-test-scheduled:
+  build-and-test:
```

Comment:
Why did you change here?

Reply:
The reason is the same as follwing thread. Let's discuss here:
https://github.com/autowarefoundation/autoware.universe/pull/297/files#r792369262